src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -130,7 +134,7 @@
      * Define the RegExp to match the relief comment that suppresses
      * the warning about a fall through.
      */
-    private Pattern reliefPattern = Pattern.compile("fallthru|falls? ?through");
+    private Pattern reliefPattern = Pattern.compile("falls?[ -]?thr(u|ough)");
```

Comment:
please create UT on this, we cover all functionality by tests.
As it affects Google style please add test to `it` part dedicated to google style coverage.

as last step (just to be 100% sure) please share diff report that can be generated by https://github.com/checkstyle/contribution/tree/master/checkstyle-tester, please review documentation (there examples of execution) and do not forget active all project in config and activate Check that you changed.

Reply:
done.

report is generating....